### ISSUE TITLE
NPM / Laravel Mix - File to import not found or unreadable: spinners.

### ISSUE THREAD

**Issue by Braunson** [franzheidl/spinners#4]
I'm trying to include the spinners lib via my `.scss` file. I've included it via NPM and when I try to compile my .scss file I get..

```
File to import not found or unreadable: spinners.
```

I can confirm it's there in `node_modules` and exists, this is a fresh project.

**Comment by franzheidl** [comment]
Have you imported it properly into your main `.scss` file?
Just the fact it is installed in your `node_modules` does not make it available in your Sass yet, you’ll have to `@import` it using the path to where it is in your node modules…

**Comment by Braunson** [comment]
@franzheidl Yeah I tried importing it as `@import "spinners";` and that didn't work.

In the end I got it working (the compiled css version) and had to include it like so:

```
@import "~spinners/main.css";
```

It's weird I've imported other libraries that auto mapped to the scss or css file and I just had to import `@import "somelib";` but for some reason with spinners I had to specify the file. Perhaps an issue with the package.json in your repo?
